Makefile

```diff
@@ -1,19 +1,35 @@
 install:
-	if [ ! -d dist ]; then virtualenv dist; fi
+	if [ ! -d dist ]; then python -m venv dist; fi
```

Comment:
I'd love to get your thoughts on venvs at some point - i.e. why `python -m venv`

Reply:
I came close to flipping this to poetry. But I use python -m venv since it's shipped in every python distribution while the virtualenv package is going to be deprecated.